Sources/Cryptor/Digest.swift

```diff
@@ -51,17 +54,28 @@ public class Digest: Updatable {
     public enum Algorithm {
 		
         /// Message Digest 2 See: http://en.wikipedia.org/wiki/MD2_(cryptography)
+        @available(iOS, introduced: 2.0, obsoleted: 13.0, message: "This function is cryptographically broken and should not be used in security contexts. Clients should migrate to SHA256 (or stronger).")
         case md2
 		
         /// Message Digest 4
+        @available(iOS, introduced: 2.0, obsoleted: 13.0, message: "This function is cryptographically broken and should not be used in security contexts. Clients should migrate to SHA256 (or stronger).")
         case md4
 		
         /// Message Digest 5
+        @available(iOS, introduced: 2.0, obsoleted: 13.0, message: "This function is cryptographically broken and should not be used in security contexts. Clients should migrate to SHA256 (or stronger).")
         case md5
 		
+        /// Message Digest 5
+        /// - NOTE: Do NOT use for cryptography, considered insecure
+        case md5_insecure
```

Comment:
If you're adding _insecure for md5 and sha1, why not md2 and md4?

It'd be cool if there was a way to group the insecure methods under ".insecure.md5"... but I'm not sure that could be done as an enum. :-/